### DISCUSSION TITLE
Provide original script used to generate voice samples

### DISCUSSION THREAD

**Discussion by Vagif12** [coqui-ai/TTS#1362] [General Q&A]
Hey!
Would it be possible to provide the original python scripts/colab notebook used to generate the voice samples as shown on:
https://erogol.github.io/ddc-samples/

Myself and others have had challenges reproducing the samples shown, and I believe that perhaps providing these samples will simplify the process of trying to reproduce them.

@erogol is this possible?

If you want to, just send it over to my email: vagal2003@gmail.com and I'll share it with others who are interested. Alternatively, a Colab file/ Google Drive link to the scripts would be super!
Thanks!

**Comment by robinsingh1** [comment]
++ am unable to repro samples
